AlgebraInLean/Basic.lean

```diff
@@ -1 +1,50 @@
-def hello := "world"
\ No newline at end of file
+import Mathlib.Tactic
+
+namespace Defs
+  class Magma (G : Type*) where
+    op : G → G → G
+
+  -- FIXME: is this actually helpful/necessary
+  attribute [always_inline] Magma.op
+
+  infixl:75 " ⋆ " => Magma.op
```

Comment:
This character looks somewhat similar to asterisk on some fonts, which is what is usually used for multiplication. It might be worth considering a different character, perhaps as a TODO/FIXME.